python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceAutomaticSpeechRecognitionPromptSchema.ts

```diff
@@ -0,0 +1,57 @@
+import { PromptSchema } from "../../utils/promptUtils";
+
+export const HuggingFaceAutomaticSpeechRecognitionPromptSchema: PromptSchema = {
+  // See https://huggingface.co/docs/transformers/v4.36.1/en/main_classes/pipelines#transformers.AutomaticSpeechRecognitionPipeline
+  // for descriptions.
+
+  // The following supported properties cannot be cleanly modeled in the schema and must be implemented programatically:
+  // tokenizer, feature_extractor, device, decoder
```

Comment:
Same question as in previous diff for this

Reply:
Another thing to note is that the completion params aren't always included in description. Ex: `binary_output` is in the completion params but not explained
<img width="945" alt="Screenshot 2024-01-09 at 12 49 41" src="https://github.com/lastmile-ai/aiconfig/assets/151060367/58dd1bdd-66aa-40a0-86f0-756936dce0d0">